db/migrate/20211213224957_fix_title_in_post.rb

```diff
@@ -0,0 +1,5 @@
+class FixTitleInPost < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :posts, :string, :title
```

Comment:
-  [ ]  OPTIONAL
The issue of `string used as column name` is not properly fixed. An error is occuring when I attempt to migrate the database. Kindly inspect the migration files accordingly and fix this issue.

**Migration error:**
![image](https://user-images.githubusercontent.com/12745474/145989297-f44012a0-c62f-40be-a28c-a65bf10d01ca.png)

Reply:
There is no migration error at my end. the data has been migrated and automatically updated inside the schema file. I think the problem is from your end. I thank you for trying to help out.

Reply:
![IssueSolved](https://user-images.githubusercontent.com/63972670/146027014-03b07333-d1a5-4dcb-b0c8-4cc4caa851e5.JPG)